### PR TITLE
Bugfix/issue 72 sort cards

### LINKED ISSUE #72: Cards should be returned sorted from the server
The server should be sorting cards by suit then value when storing them for the user

### PR DESCRIPTION
**Pull request by creisle**
Feature
- sort player cards when setting them in the db

BugFix
- close db connection on tests done

Style
- fix some extra whitespace

### CODE REVIEW

**Comment by areisle** [comment]
I don't think this sorts the initial cards you send me

**Review by areisle — CHANGES_REQUESTED** [review]
in server/index when you shuffle the deck and deal the cards, you return those not the cards you write to redis no?

File: server/db/round.ts [hunk 1]
```diff
@@ -326,12 +326,40 @@ const getPlayerCards = async (redis: Redis, gameId: string, playerId: string, ro
...
+        return suitOrder[card1.suit] - suitOrder[card2.suit];
+    };
+    return cards.sort(compareCards)
+}
+
+
 const setPlayerCards = async (redis: Redis, gameId: string, playerId: string, roundNumber: number, cards: Card[]) => {
     await redis.del(`${gameId}-r${roundNumber}-p${playerId}-cards`);
+
     if (cards.length) {
```
Comment (areisle) [inline server/db/round.ts:358]: not part of this PR, but just curious; why do you check the length?

**Review by areisle — APPROVED** [review]

File: server/db/round.ts (hunk 1 above)
Comment (creisle) [reply server/db/round.ts:358]: b/c redis will throw an error if I try to push nothing
